spec/forms/support_interface/country_form_spec.rb

```diff
@@ -192,4 +192,48 @@
       end
     end
   end
+
+  describe "#for_existing_country" do
+    subject(:form) { described_class.for_existing_country(country) }
+
+    let(:country) { create(:country) }
+
+    it "returns a CountryForm" do
```

Comment:
Can we improve the text here?

Maybe something like "returns a instance of a CountryForm for the country"